<?xml version='1.0' encoding='UTF-8'?><ResultsSession buildId="Default Project" climode="true" endTime="2024-10-16T14:59:04+08:00" execEnv="win32_x86_64" hasViols="true" id="1729402026912" lang="en_US" machine="Test" project="Default Project" tag="Demo Configuration" time="2024-10-16T14:59:03+08:00" toolDispName="SOAtest" toolId="soatest" toolName="SOAtest" toolVer="2024.1.3.202409292000" user="test">
   <TestConfig machine="Test" name="Demo Configuration" pseudoUrl="soatest.builtin://Demo Configuration" user="test"/>
   <Authors>
      <Author id="dev1" name="test"/>
   </Authors>
   <Analyzers>
      <Analyzer id="com.parasoft.soavirt.desktop.functional.analyzer" name="Parasoft Functional Tests Analyzer" ver="10.7.0.202409292000"/>
   </Analyzers>
   <VersionInfos>
      <StorageInfo ownerId="" resultId="ISetupResult" ver="1" ver10x="1"/>
      <StorageInfo ownerId="" resultId="IExecutionViolation" ver="1" ver10x="1"/>
      <StorageInfo ownerId="com.parasoft.xtest.checkers.api.standards" resultId="IDupCodeViolation" ver="2" ver10x="1"/>
      <StorageInfo ownerId="com.parasoft.xtest.checkers.api.standards" resultId="IFlowAnalysisViolation" ver="2" ver10x="1"/>
      <StorageInfo ownerId="" resultId="IFunctionalViolation" ver="1" ver10x="1"/>
      <StorageInfo ownerId="" resultId="IChangeImpactViolation" ver="1" ver10x="1"/>
      <StorageInfo ownerId="com.parasoft.xtest.checkers.api.standards" resultId="ICodingStandardsViolation" ver="2" ver10x="1"/>
      <StorageInfo ownerId="" resultId="IUnitTestViolation" ver="1" ver10x="1"/>
   </VersionInfos>
   <Scope>
      <Repositories/>
      <Locations>
         <Loc hash="-2030685509" locRef="1" projId="TestAssets" projPath="/TestAssets" project="TestAssets" resProjPath="PB-WebUI-v3.tst" totLns="331" uri="file:/C:/soatest/TestAssets/PB-WebUI-v3.tst"/>
      </Locations>
   </Scope>
   
  <SetupProblems/>

   <Exec ownerId="com.parasoft.xtest.checkers.api.execution" time="0:00:06">
      
      <AssocUrls>
         <Tag disp="Requirement" name="req"/>
         <Tag disp="Feature request" name="fr"/>
         <Tag disp="Problem report" name="pr"/>
         <Tag disp="Task" name="task"/>
         <Tag disp="Asset" name="asset"/>
         <Tag disp="Test" name="test"/>
      </AssocUrls>
   
      
      <Categories>
         <CategoryGroup authTot="204;" authUrg="0;" desc="Functional Test Problems" id="GROUP.FUNC.FAIL" label="Fix Functional Test Problems" name="Functional Test Problems" total="204">
            <Category authTot="101;" authUrg="0;" id="FUNC.FAIL" name="Functional Failures" short="functional failure" total="101">
               <Type authTot="101;" authUrg="0;" name="unknown" total="101"/>
            </Category>
            <Category authTot="0;" authUrg="0;" id="XML.VALID.ERROR" name="XML Validation Errors" short="XML validation error" total="0"/>
            <Category authTot="103;" authUrg="0;" id="MISC.ERR" name="Miscellaneous Errors" short="miscellaneous error" total="103">
               <Type authTot="103;" authUrg="0;" name="unknown" total="103"/>
            </Category>
         </CategoryGroup>
         <CategoryGroup authTot="0;" authUrg="0;" desc="Change Impact" id="GROUP.CHNG.IMP" label="Review Change Impact" name="Change Impact" total="0">
            <Category authTot="0;" authUrg="0;" id="CHNG.IMP" name="Change Impact" short="change impact" total="0"/>
         </CategoryGroup>
      </Categories>
      <Projects>
         <Project err="0" fail="5" id="TestAssets" name="TestAssets" pass="0" total="5">
            <CategoryGroupStats id="GROUP.FUNC.FAIL" total="204"/>
            <CategoryGroupStats id="GROUP.CHNG.IMP" total="0"/>
         </Project>
      </Projects>
      
  <APICoverage>
    <service type="Unknown Service">
      <resource id="http://localhost:8090/parabank/services/ParaBank">
        <operation name="cleanDB">
          <test-ref id="wk:///TestAssets/PB-ParabankServices-v3.tst#0000000000#0000000428" pentest="false"/>
          <test-ref id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000028" pentest="false"/>
        </operation>
      </resource>
    </service>
  </APICoverage>

      
  <TrafficData>
    <Traffic serverResTime="0" resBody="" reqBody="&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?>&#xa;&lt;SOAP-ENV:Envelope xmlns:SOAP-ENV=&quot;http://schemas.xmlsoap.org/soap/envelope/&quot; xmlns:xsd=&quot;http://www.w3.org/2001/XMLSchema&quot; xmlns:xsi=&quot;http://www.w3.org/2001/XMLSchema-instance&quot;>&#xa; &lt;SOAP-ENV:Body>&#xa;  &lt;login xmlns=&quot;http://service.parabank.parasoft.com/&quot;>&#xa;   &lt;username>john&lt;/username>&#xa;   &lt;password>demo&lt;/password>&#xa;  &lt;/login>&#xa; &lt;/SOAP-ENV:Body>&#xa;&lt;/SOAP-ENV:Envelope>" resMIMEType="text/plain" reqTime="0" reqMIMEType="text/xml" tfcRef="1"/>
  </TrafficData>
      
  <EnvData/>
      <FuncAPISecurity pentest="false">
         
  <Categories/>
         
  <Attacks/>
      </FuncAPISecurity>
      <Axe scan="false">
         
  <AxeRules/>
      </Axe>
      
  <ExecViols>
    <FuncViol msg="Invalid WSDL. Connection refused: getsockopt" loc="/TestAssets/PB-WebUI-v3.tst" ln="-1" sev="1" auth="test" locRef="1" violationDetails="" startPos="-1" endPos="-1" fatal="false" analyzer="com.parasoft.soavirt.desktop.functional.analyzer" cat="MISC.ERR" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000028"/>
    <FuncViol msg="Connection refused: connect" loc="/TestAssets/PB-WebUI-v3.tst" ln="-1" sev="1" auth="test" locRef="1" violationDetails="A connection error has been detected.  This may occur if a test has timed out while the client was still attempting to send or receive data." startPos="-1" endPos="-1" fatal="true" analyzer="com.parasoft.soavirt.desktop.functional.analyzer" cat="FUNC.FAIL" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000028" tfcRef="1"/>
    <FuncViol msg="Connection refused: connect" loc="/TestAssets/PB-WebUI-v3.tst" ln="-1" sev="1" auth="test" locRef="1" violationDetails="com.parasoft.api.tool.CustomToolException: Connection refused: connect&#xd;&#xa;&#x9;at StopCoverageSession.execute(StopCoverageSession.java:51)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;Caused by: java.net.ConnectException: Connection refused: connect&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect0(Native Method)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:579)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:568)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:639)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:588)&#xd;&#xa;&#x9;at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.&lt;init>(HttpClient.java:281)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1311)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1244)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1130)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1059)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1689)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1613)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3313)&#xd;&#xa;&#x9;at CoverageAgentController.getResponse(CoverageAgentController.java:238)&#xd;&#xa;&#x9;at CoverageAgentController.execute(CoverageAgentController.java:88)&#xd;&#xa;&#x9;at StopCoverageSession.execute(StopCoverageSession.java:47)&#xd;&#xa;&#x9;... 6 more&#xd;&#xa;[Nested Exception] java.net.ConnectException: Connection refused: connect&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect0(Native Method)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:579)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:568)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:639)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:588)&#xd;&#xa;&#x9;at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.&lt;init>(HttpClient.java:281)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1311)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1244)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1130)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1059)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1689)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1613)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3313)&#xd;&#xa;&#x9;at CoverageAgentController.getResponse(CoverageAgentController.java:238)&#xd;&#xa;&#x9;at CoverageAgentController.execute(CoverageAgentController.java:88)&#xd;&#xa;&#x9;at StopCoverageSession.execute(StopCoverageSession.java:47)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;" startPos="-1" endPos="-1" fatal="true" analyzer="com.parasoft.soavirt.desktop.functional.analyzer" cat="FUNC.FAIL" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000087"/>
    <FuncViol msg="Connection refused: connect" loc="/TestAssets/PB-WebUI-v3.tst" ln="-1" sev="1" auth="test" locRef="1" violationDetails="com.parasoft.api.tool.CustomToolException: Connection refused: connect&#xd;&#xa;&#x9;at StartCoverageSession.execute(StartCoverageSession.java:51)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;Caused by: java.net.ConnectException: Connection refused: connect&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect0(Native Method)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:579)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:568)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:639)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:588)&#xd;&#xa;&#x9;at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.&lt;init>(HttpClient.java:281)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1311)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1244)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1130)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1059)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1689)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1613)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3313)&#xd;&#xa;&#x9;at CoverageAgentController.getResponse(CoverageAgentController.java:238)&#xd;&#xa;&#x9;at CoverageAgentController.execute(CoverageAgentController.java:88)&#xd;&#xa;&#x9;at StartCoverageSession.execute(StartCoverageSession.java:47)&#xd;&#xa;&#x9;... 6 more&#xd;&#xa;[Nested Exception] java.net.ConnectException: Connection refused: connect&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect0(Native Method)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:579)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:568)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:639)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:588)&#xd;&#xa;&#x9;at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.&lt;init>(HttpClient.java:281)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1311)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1244)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1130)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1059)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1689)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1613)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3313)&#xd;&#xa;&#x9;at CoverageAgentController.getResponse(CoverageAgentController.java:238)&#xd;&#xa;&#x9;at CoverageAgentController.execute(CoverageAgentController.java:88)&#xd;&#xa;&#x9;at StartCoverageSession.execute(StartCoverageSession.java:47)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;" startPos="-1" endPos="-1" fatal="true" analyzer="com.parasoft.soavirt.desktop.functional.analyzer" cat="FUNC.FAIL" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000088"/>
    <FuncViol msg="Connection refused: connect" loc="/TestAssets/PB-WebUI-v3.tst" ln="-1" sev="1" auth="test" locRef="1" violationDetails="com.parasoft.api.tool.CustomToolException: Connection refused: connect&#xd;&#xa;&#x9;at StartCoverageTest.execute(StartCoverageTest.java:51)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;" startPos="-1" endPos="-1" fatal="true" analyzer="com.parasoft.soavirt.desktop.functional.analyzer" cat="FUNC.FAIL" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000001#0000000002#0000000091" tcId="Chrome"/>
    <FuncViol msg="Connection refused: connect" loc="/TestAssets/PB-WebUI-v3.tst" ln="-1" sev="1" auth="test" locRef="1" violationDetails="com.parasoft.api.tool.CustomToolException: Connection refused: connect&#xd;&#xa;&#x9;at StopCoverageSession.execute(StopCoverageSession.java:51)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;Caused by: java.net.ConnectException: Connection refused: connect&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect0(Native Method)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:579)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:568)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:639)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:588)&#xd;&#xa;&#x9;at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.&lt;init>(HttpClient.java:281)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1311)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1244)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1130)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1059)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1689)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1613)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3313)&#xd;&#xa;&#x9;at CoverageAgentController.getResponse(CoverageAgentController.java:238)&#xd;&#xa;&#x9;at CoverageAgentController.execute(CoverageAgentController.java:88)&#xd;&#xa;&#x9;at StopCoverageSession.execute(StopCoverageSession.java:47)&#xd;&#xa;&#x9;... 6 more&#xd;&#xa;[Nested Exception] java.net.ConnectException: Connection refused: connect&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect0(Native Method)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:579)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.Net.connect(Net.java:568)&#xd;&#xa;&#x9;at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:639)&#xd;&#xa;&#x9;at java.base/java.net.Socket.connect(Socket.java:588)&#xd;&#xa;&#x9;at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.&lt;init>(HttpClient.java:281)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)&#xd;&#xa;&#x9;at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1311)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1244)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1130)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1059)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1689)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1613)&#xd;&#xa;&#x9;at java.base/sun.net.www.protocol.http.HttpURLConnection.getHeaderFields(HttpURLConnection.java:3313)&#xd;&#xa;&#x9;at CoverageAgentController.getResponse(CoverageAgentController.java:238)&#xd;&#xa;&#x9;at CoverageAgentController.execute(CoverageAgentController.java:88)&#xd;&#xa;&#x9;at StopCoverageSession.execute(StopCoverageSession.java:47)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:267)&#xd;&#xa;&#x9;at webtool.extension.tool.CustomTool$ApplyTool.call(CustomTool.java:1)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xd;&#xa;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xd;&#xa;&#x9;at java.base/java.lang.Thread.run(Thread.java:840)&#xd;&#xa;" startPos="-1" endPos="-1" fatal="true" analyzer="com.parasoft.soavirt.desktop.functional.analyzer" cat="FUNC.FAIL" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000089"/>
  </ExecViols>

      
   <ExecutedTestsDetails functional="true" changeImpact="false" type="FT">
      <Total fail="5" total="5" name="Passed / Total" time="0:00:06.250" authErr="0;" authFail="5;">
         <Project fail="5" total="5" name="TestAssets" time="0:00:06.250" authErr="0;" id="TestAssets" authFail="5;">
            <TestSuite ln="0" locRef="1" root="true" name="PB-WebUI-v3.tst" id="wk:///TestAssets/PB-WebUI-v3.tst" fail="5" total="5" time="0:00:00.107" authErr="0;" authFail="5;">
               <TestSuite ln="0" locRef="1" name="Test Suite: Test Suite" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000" fail="5" total="5" time="0:00:00.107" authErr="0;" authFail="5;">
                  <TestSuite ln="0" locRef="1" name="Set-Up Tests" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#$$SetupIndex&amp;&amp;" fail="3" total="3" time="0:00:00.075" authErr="0;" authFail="3;">
                     <Test ln="0" auth="test" locRef="1" name="Set-Up 1: cleanDB" startTime="1729061937612" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000028" status="fail" fail="1" total="1" time="0:00:00.039" authErr="0;" authFail="1;"/>
                     <Test ln="0" auth="test" locRef="1" name="Set-Up 2: Stop Coverage Session" startTime="1729061937651" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000087" status="fail" fail="1" total="1" time="0:00:00.020" authErr="0;" authFail="1;"/>
                     <Test ln="0" auth="test" locRef="1" name="Set-Up 3: Start Coverage Session" startTime="1729061937672" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000088" status="fail" fail="1" total="1" time="0:00:00.016" authErr="0;" authFail="1;"/>
                  </TestSuite>
                  <TestSuite ln="0" locRef="1" name="Test Suite: Web Scenarios" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000001" fail="1" total="1" time="0:00:00.018" authErr="0;" authFail="1;">
                     <TestSuite ln="0" locRef="1" name="Scenario: Basic UX Workflow" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000001#0000000002" fail="1" total="1" time="0:00:00.018" authErr="0;" authFail="1;">
                        <Test ln="0" auth="test" locRef="1" name="Test 1: Start Coverage Test: Web UI/UX" startTime="0" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000001#0000000002#0000000091" status="fail" fail="1" total="1" time="0:00:00.018" authErr="0;" authFail="1;">
                           <assoc id="PAR-32" tag="req"/>
                           <assoc id="PAR-4" tag="req"/>
                           <TestCase auth="test" name="Test 1: Start Coverage Test: Web UI/UX: Chrome" startTime="1729061937720" testId="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000001#0000000002#0000000091" id="Chrome" status="fail" fail="1" total="1" time="0:00:00.018" authErr="0;" authFail="1;"/>
                        </Test>
                     </TestSuite>
                  </TestSuite>
                  <TestSuite ln="0" locRef="1" name="Tear-Down Tests" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#$$TeardownIndex&amp;&amp;" fail="1" total="1" time="0:00:00.014" authErr="0;" authFail="1;">
                     <Test ln="0" auth="test" locRef="1" name="Tear-Down 1: Stop Coverage Session 2" startTime="1729061937741" id="wk:///TestAssets/PB-WebUI-v3.tst#0000000000#0000000089" status="fail" fail="1" total="1" time="0:00:00.014" authErr="0;" authFail="1;"/>
                  </TestSuite>
               </TestSuite>
            </TestSuite>
         </Project>
      </Total>
   </ExecutedTestsDetails>

      
  <FuncTestDetails/>
   </Exec>
   <Statistics>
      
   </Statistics>
</ResultsSession>